packages/time/src/types.d.ts

```diff
@@ -190,6 +211,10 @@ export interface TimerWaker {
   wake: (timestamp: TimestampRecord) => void;
 }
 
+/**
+ * Provides the ability to schedule wake() calls repeatedly at a regular
+ * interval, or to disable all future use of this TimerRepeater.
```

Comment:
nit, maybe add "created by the old `makeRepeater()` API".. which is deprecated (but probably not visibly enough), new code should use `repeatAfter()`, which doesn't have a control object and doesn't require a second `schedule` step

Reply:
done